frontend/src/book-read/components/reviews.component.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useFetchReviewsByBookQuery } from '../store/apis/review';
+import { SHARED_CONFIG } from '../../shared';
+import { ReviewPaginatedList } from '../../shared/components/lists/reviews-paginated-list.jsx';
```

Comment:
That's a long import. It is this way, because it was not re-exported from index.js at "shared" module.
Also, the file name of reviews-paginated-list.jsx lacks a ".component" part.

Reply:
Fixed